pkg/apis/kubermatic/v1/etcd_backup_config.go

```diff
@@ -46,11 +46,12 @@ const (
 // +kubebuilder:subresource:status
 // +kubebuilder:printcolumn:JSONPath=".metadata.creationTimestamp",name="Age",type="date"
 
-// EtcdBackupConfig specifies a add-on.
+// EtcdBackupConfig specifies an add-on.
```

Comment:
This was always a terribly misleading doc string. KKP already has a concept of "addons" and etcd backups have _nothing_ to do with that. A better description would be something along the lines of

> EtcdBackupConfig describes how snapshots of user cluster etcds should be performed. Each user cluster automatically gets a default EtcdBackupConfig in its cluster namespace.